src/review-todo.js

```diff
@@ -0,0 +1,212 @@
+let isDate = require("date-fns/isDate");
+
+function todo(todoName = "New Todo") {
+	let title = todoName,
+		description = "",
+		urgency = 0,
+		isDone = false,
+		dueDate = todoDate(),
+		scheduleDate = todoDate(),
+		tags = tagList();
+
+	let todoObj = Object.create(todo.proto);
+	Object.assign(todoObj, {
+		title,
+		description,
+		isDone,
+		urgency,
+		dueDate,
+		scheduleDate,
+		tags,
+	});
+
+	return todoObj;
+}
+
+todo.proto = {};
+
+function project(projectName = "New Project") {
+	let proto = Object.assign(project.proto, { isChild });
+	let projectObj = Object.create(proto);
```

Comment:
Same idea with constructors.

I think on line 29 you're creating an Object that adds the `isChild` method to the project prototype. Instead,
could you write this function as a constructor and add `isChild` to the `Project` prototype?

The constructor function looks the same as above, but to add this new method you'll do something like this:
```
Project.prototype = {
  isChild: function(otherProject) { ... method definition }
}
```
It seemed really weird to me when I first learned it, but basically, we're saying the prototype is an object with our method name as the key and the behavior as the body.